index.js

```diff
@@ -1,12 +1,24 @@
 'use strict';
 var got = require('got');
 var registryUrl = require('registry-url');
+var rc = require('rc');
 
 module.exports = function (name, version) {
-	var url = registryUrl(name.split('/')[0]) +
+	var scope = name.split('/')[0];
+	var url = registryUrl(scope) +
 		encodeURIComponent(name).replace(/^%40/, '@');
+	var npmrc = rc('npm');
+	var token = scope ? npmrc[scope + ':_authToken'] : npmrc['//registry.npmjs.org/:_authToken'];
```

Comment:
How about now?

The `npmrc[scope + ':_authToken']` there for people that have scoped privately hosted modules.